docs/en-US/component/cascader.md

```diff
@@ -142,96 +142,103 @@ cascader/panel
+| collapse-tags                          | whether to collapse tags in multiple selection mode                                                                                                                              | ^[boolean]                                                    | —       |
+| collapse-tags-tooltip                  | whether show all selected tags when mouse hover text of collapse-tags. To use this, `collapse-tags` must be true                                                                 | ^[boolean]                                                    | false   |
+| separator                              | option label separator                                                                                                                                                           | ^[string]                                                     | ' / '   |
+| filterable                             | whether the options can be searched                                                                                                                                              | ^[boolean]                                                    | —       |
+| filter-method                          | customize search logic, the first parameter is `node`, the second is `keyword`, and need return a boolean value indicating whether it hits.                                      | ^[Function]`(node: CascaderNode, keyword: string) => boolean` | —       |
+| debounce                               | debounce delay when typing filter keyword, in milliseconds                                                                                                                       | ^[number]                                                     | 300     |
+| before-filter                          | hook function before filtering with the value to be filtered as its parameter. If `false` is returned or a `Promise` is returned and then is rejected, filtering will be aborted | ^[Function]`(value: string) => boolean`                       | —       |
+| popper-class                           | custom class name for Cascader's dropdown                                                                                                                                        | ^[string]                                                     | ''      |
+| teleported                             | whether cascader popup is teleported                                                                                                                                             | ^[boolean]                                                    | true    |
+| popper-append-to-body<DeprecatedTag /> | whether to append the popper menu to body. If the positioning of the popper is wrong, you can try to set this prop to false                                                      | ^[boolean]                                                    | true    |
```

Comment:
We should update this to our new syntax, refer to #11580

Reply:
done.